.github/workflows/kaito-e2e.yml

```diff
@@ -17,9 +17,19 @@ permissions:
 
 jobs:
   run-e2e:
+    strategy:
+      fail-fast: false
+      matrix:
+        suite: [ gpuprovisioner ]
```

Comment:
rename the variable name from suite to nodeprovisioner?

Reply:
Will be fixed on the upcoming PR.

Reply:
Fixed in #570